content/basics/vimmodes/insert.md

```diff
@@ -4,4 +4,18 @@ date =  2017-10-30T10:48:58-04:00
 weight = 6
 +++
 
-Insert Mode
+Insert Mode is where you can write/edit your documments. There are some keys to enter in this mode, let's see them:  
```

Comment:
Insert Mode is where you can write/edit your documments. From **Normal/Command** mode, you can use one of the following keys to switch to **Insert** mode: